sklearn/metrics/_plot/cap_curve.py

```diff
@@ -0,0 +1,274 @@
+from ...utils._plotting import _BinaryClassifierCurveDisplayMixin
+import numpy as np
+
+
+class CumulativeAccuracyDisplay(_BinaryClassifierCurveDisplayMixin):
+    """Cumulative Accuracy Profile (CAP) Curve visualization.
+
+    It is recommended to use
+    :func:`~sklearn.metrics.CumulativeAccuracyDisplay.from_estimator` or
+    :func:`~sklearn.metrics.CumulativeAccuracyDisplay.from_predictions` to create
+    a :class:`~sklearn.metrics.CumulativeAccuracyDisplay`. All parameters are
+    stored as attributes.
+
+    Read more in the :ref:`User Guide <visualizations>`.
+
+    Parameters
+    ----------
+    cumulative_true_positives : ndarray
```

Comment:
I wonder how to change that name so that it would make sense both for classification problems and for positive regression problems (e.g. as in a Lorenz curve).

Any opinion @lorentzenchr?

Reply:
`y_true_cumulative` and mention in parameter docstring that for binary classification, the true positive has a value of 1.